econml/panel/utilities.py

```diff
@@ -0,0 +1,39 @@
+
+import numpy as np
+
+
+def long(x):
+    """
+    Reshape panel data to long format, i.e. (n_units * n_periods, d_x) or (n_units * n_periods,)
+
+    Parameters
+    ----------
+    x : array-like
+        Panel data in wide format
```

Comment:
I think it could be helpful to be more specific about what we mean by wide format. 

Does this mean x can be either (n_units, n_periods, d_x) shaped or (n_units, n_periods)?

The two examples I see for wide format in the `wide` docstring are 2-dim arrays, so it's not clear to me why we need to deal with the case when x is not 2 dimensions.